src/ProductDetail.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+function ProductDetail(props) {
```

Comment:
This looks like you were experimenting with things trying to figure out how props works, which is good.
But you should probably go back and  try to code it the way the readme describes.
You can't hard code 5 product details, that is not realistic to how real apps work.
Where is your use of the map function